README.md

```diff
@@ -1,2 +1,49 @@
-# moodster-app
+# Moodster
 Moodster Application Project
+
+# Setting up the Development Environment
+
+Go to https://facebook.github.io/react-native/docs/getting-started.html, and select the **React Native CLI Quickstart** tab as this project doesn't use Expo. **Follow these instructions to setup your development environment.**
+
+### Windows/Linux
+
+You will need Node, the React Native command line interface, Python2, a JDK, and Android Studio.
```

Comment:
What is the requirement on python2? If it is for the moodster-api then I would change that to python3 as I will be adding changes that will not work in python2.

Reply:
Python2 is a requirement for react-native as far as I'm aware, nothing to do with the api.

Reply:
So you may want to change the python2 requirement to a windows specific requirement.